ui/packages/consul-ui/package.json

```diff
@@ -58,7 +58,6 @@
     "@babel/plugin-proposal-object-rest-spread": "^7.5.5",
     "@docfy/ember": "^0.4.1",
     "@ember/optional-features": "^2.0.0",
-    "@ember/render-modifiers": "^1.0.2",
```

Comment:
is this not used anymore? you might need to just bump the version to get rid of deprecation

Reply:
Yeah we should not rely on anything that is not explicitly listed as dependency, it's pretty dangerous